pom.xml

```diff
@@ -39,6 +39,11 @@
         <artifactId>pdfbox</artifactId>
         <version>${pdfbox.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.apache.pdfbox</groupId>
+        <artifactId>pdfbox-tools</artifactId>
+        <version>${pdfbox.version}</version>
+      </dependency>
```

Comment:
This depends on Picocli, don't we need to depend on quarkus-picocli too ?

Reply:
good point let me double check

Reply:
I actually wonder if we should Exclude Picocoli from this dependency.  I think they are just using it for some tools and not any CLI

Reply:
Yeah, I think that's better

Reply:
Excluded!